api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -1028,7 +1028,7 @@ public class ApiConstants {
     public static final String VM_SCHEDULE_ID = "vmscheduleid";
 
     public static final String VM_SCHEDULE_IDS = "vmscheduleids";
-
+    public static final String VM_SCHEDULE_STATE = "state";
```

Comment:
does state not yet exist as generic constant? it would be beating the purpose to create two constants with the same value.

Reply:
Fixed.